spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -489,6 +489,10 @@ def when_I_unschedule_publishing
   def and_the_step_by_step_is_not_editable
     then_there_should_be_no_reorder_steps_tab
 
+    when_I_edit_the_step_by_step_page
+    then_I_can_see_the_step_by_step_details
+    and_I_cannot_the_step_by_step_details
```

Comment:
Typo: I think this test name is missing a word. Perhaps it should be `and_I_cannot_edit_the_step_by_step_details`?

Reply:
🤦‍♂ It's missing the verb. Updated now